ui/preferences/src/main/res/xml/preferences_autodownload.xml

```diff
@@ -1,13 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<PreferenceScreen
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:search="http://schemas.android.com/apk/com.bytehamster.lib.preferencesearch">
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <de.danoeh.antennapod.ui.preferences.preference.MasterSwitchPreference
```

Comment:
This was the only use of `MasterSwitchPreference`. I think the class can therefore be removed.